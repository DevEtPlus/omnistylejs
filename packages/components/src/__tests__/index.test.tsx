import React from "react";
import { View } from "react-native";

import renderer from "react-test-renderer";

function toJson(component: renderer.ReactTestRenderer) {
  const result = component.toJSON();

  expect(result).toBeDefined();
  expect(result).not.toBeInstanceOf(Array);

  return result as renderer.ReactTestRendererJSON;
}

describe("@omnistylejs/components", () => {
  it("Basic View render", () => {
    const component = renderer.create(<View />);

    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});
