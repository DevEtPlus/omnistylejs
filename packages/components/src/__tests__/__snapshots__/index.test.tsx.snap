// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@omnistylejs/components Basic View render 1`] = `<View />`;
